test/easyconfigs/easyconfigs.py

```diff
@@ -1027,7 +1022,7 @@ def test_sanity_check_paths(self):
         """Make sure specified sanity check paths adher to the requirements."""
 
         for ec in self.parsed_easyconfigs:
-            ec_scp = ec['ec'].get_ref('sanity_check_paths')
+            ec_scp = ec['ec'].get('sanity_check_paths', resolve=False)
```

Comment:
This and below is from the resolved conflict. Kept it as IMO `resolve=False` makes the intent clearer